dev/cnf/oss_dependencies.maven

```diff
@@ -312,6 +312,7 @@ org.eclipse.microprofile.metrics:microprofile-metrics-api:1.0
 org.eclipse.microprofile.metrics:microprofile-metrics-api:1.1.1
 org.eclipse.microprofile.metrics:microprofile-metrics-api:2.0.0
 org.eclipse.microprofile.metrics:microprofile-metrics-api:2.1.0
```

Comment:
Do you want to remove the 2.1 dependency if we're not using it?

Reply:
Will remove all mpMetrics-2.1 references in separate PR